README.rst

```diff
@@ -141,4 +141,4 @@ We happily welcome contributions. Please see the `Contributing <https://github.c
 
    All content found in the Spinal Cord Toolbox repository and spinalcordtoolbox.com website, including: text, images, audio, or other formats were created for informational purposes only. The content is not intended to be a substitute for professional medical advice, diagnosis, or treatment. Always seek the advice of your physician or other qualified health provider with any questions you may have regarding a medical condition. Never disregard professional medical advice or delay in seeking it because of something you have read on this website.
 
-   If you think you may have a medical emergency, call your doctor, go to the emergency department, or call your local emergency number immediately. Spinal Cord Toolbox does not recommend or endorse any specific tests, physicians, products, procedures, opinions, or other information that may be mentioned on spinalcordtoolbox.com. Reliance on any information provided by spinalcordtoolbox.com, Spinal Cord Toolbox contributors, contracted writers, or medical professionals presenting content for publication to spinalcordtoolbox.com is solely at your own risk.
\ No newline at end of file
+   If you think you may have a medical emergency, call your doctor, go to the emergency department, or call your local emergency number immediately. Spinal Cord Toolbox does not recommend or endorse any specific tests, physicians, products, procedures, opinions, or other information that may be mentioned on spinalcordtoolbox.com. Reliance on any information provided by spinalcordtoolbox.com, Spinal Cord Toolbox contributors, contracted writers, or medical professionals presenting content for publication to spinalcordtoolbox.com is solely at your own risk.
```

Comment:
I scanned this part, but couldn't see any changes. I think it's just a newline change? I can revert that to minimize the changes from this PR.

Reply:
Done in [fe27f88](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4014/commits/fe27f8830cfc92a70592439d30ba97d1a5f1906a).